src/base.pug

```diff
@@ -67,6 +67,8 @@ html.no-js(lang="en-GB")
     meta(name="twitter:site" content="@we_are_leaf")
     meta(name="twitter:creator" content="@we_are_leaf")
     link(rel="stylesheet", href="/assets/styles/main.css")
+    link(href='https://fonts.googleapis.com/css2?family=Playfair+Display:ital@1&display=swap' rel='stylesheet')
```

Comment:
Are we using this on every page? If not, should we configure it to only be included on the ones its needed on?

Reply:
Good spot. Currently just the blog pages, so I'll cap it to there.